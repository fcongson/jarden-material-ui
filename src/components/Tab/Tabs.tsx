import { Tabs as MaterialTabs, ThemeProvider } from '@material-ui/core'
import * as React from 'react'
import { theme } from '../../styles/material'

interface TabsProps {
  value: string
  onChange: (value: string) => void
}

export const Tabs: React.FC<TabsProps> = ({ value, onChange, children, ...rest }) => (
  <ThemeProvider theme={theme}>
    <MaterialTabs
      value={value}
      onChange={(event, value) => {
        if (!!onChange) onChange(value)
      }}
      {...rest}>
      {children}
    </MaterialTabs>
  </ThemeProvider>
)
