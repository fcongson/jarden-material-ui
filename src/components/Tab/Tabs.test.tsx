import { shallow } from 'enzyme'
import * as React from 'react'
import { Tabs } from './Tabs'

describe('Tabs', () => {
  it('Renders', () => {
    const tabs = shallow(<Tabs value='value' onChange={value => {}} />)
    expect(tabs).toMatchSnapshot()
  })
})
