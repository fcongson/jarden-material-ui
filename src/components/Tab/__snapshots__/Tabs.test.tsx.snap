// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs Renders 1`] = `ShallowWrapper {}`;
